hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitsFilters.tsx

```diff
@@ -274,14 +292,14 @@ export const OrgUnitFilters: FunctionComponent<Props> = ({
                     <>
                         <InputComponent
                             type="select"
-                            disabled={isFetchingOrgunitTypes}
+                            disabled={isFetchingOrgUnitTypes}
                             keyValue="version"
                             onChange={handleChange}
                             value={sourceVersionId}
                             label={MESSAGES.sourceVersion}
                             options={versionsDropDown}
                             clearable={false}
-                            loading={isFetchingOrgunitTypes}
```

Comment:
@bmonjoie is this means that the displaying of sources is related to the loading of the orgUnitTypes?

Reply:
The only thing I did here is move the logic of fetching OrgUnitTypes within the component instead of outside it like it used to be. You see a line change here because I have fixed the case: `Orgunit` became `OrgUnit`.
I have no idea why the two were linked and I haven't changed that logic.